frontend/src/pages/dashboard/dashboard.js

```diff
@@ -75,8 +75,9 @@ export default function CreateNewTrainingPage() {
               </Typography>
             
             </CardContent>
-            <CardActions>
-              <Button size="small">View Training</Button>
+            <CardActions><Link className={classes.underline} to={`/course/${course._id}`}>
+                  <Button variant="outlined" color="secondary">View Training</Button>
+              </Link>
```

Comment:
Can you please make sure this button matches the style of the LaunchXR. It looks odd when they are different